examples/python/NeuralAssistedStereo/neural_assisted_stereo.py

```
@@ -58,7 +24,7 @@ def showDepth(depthFrame, windowName="Depth", minDistance=500, maxDistance=5000,
         pipeline.start()
         while pipeline.isRunning():
             disparity = disparityQueue.get()
-            showDepth(disparity.getCvFrame(), minDistance=100, maxDistance=6000, useLog=False)
+            cv.imshow("Depth", dai.colorizeDepthFrame(disparity, 500, 12000, cv.COLORMAP_TURBO, False).getCvFrame())
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)